### PR TITLE
big things happening

### PR DESCRIPTION
**Pull request by bellettichloe**

### CODE REVIEW

File: src/routes/create-outfit/+page.svelte [hunk 1]
```diff
@@ -70,58 +85,63 @@
...
 					</div>
-			
+					<!-- Save Outfit Section -->
 					<div class="save_outfit">
-						<div style='display:flex; justify-content:center; padding:10px;'>
-							<div style="display:flexbox; width :50%;">
-								<TextInput placeholder="Enter Outfit Name" bind:value={outfitName} />
+						<div>
+							<div style="width: 500px;">
+								<TextInput label="Enter Outfit Name" bind:value={outfitName} />
```
Comment (Nate-Louder) [inline src/routes/create-outfit/+page.svelte]:
```suggestion
								<TextInput label="Enter Outfit Name" required bind:value={outfitName} />
```

**Review by Nate-Louder — APPROVED** [review]
JimCarreySaluteGIF

**Review by Nate-Louder — APPROVED** [review]
huh

**Comment by bellettichloe** [comment]
someone please merge my pull request. I worked really really hard on it and I promise I didn't sneak any malware into src. This is a threat.

**Review by madicoulson — APPROVED** [review]
you are the best coder in the world.

**Pull request merged**